cellprofiler/setting.py

```diff
@@ -1709,12 +1709,24 @@ def shape(self, value):
 
     @property
     def size(self):
-        return int(self.value_text.split(",")[1])
+        _, size = self.value_text.split(",")
```

Comment:
@0x00b1 In the size setter (here)[https://github.com/CellProfiler/CellProfiler/blob/a2b23730cd51e9d43816e7fdaf1fcea42205ac2a/cellprofiler/setting.py#L1716-L1718]

```python
@size.setter
def size(self, value):
    self.value_text = ",".join((self.shape, str(value)))
```